edge-modules/metrics-collector/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 1.0.8 (2022-05-24)
+
+### Bug Fixes
+- Update Base Images to address Microsoft .NET Security Updates for [CVE-2022-23267](https://msrc.microsoft.com/update-guide/vulnerability/CVE-2022-23267), [CVE-2022-29117](https://github.com/dotnet/announcements/issues/220), [CVE-2022-29145](https://msrc.microsoft.com/update-guide/vulnerability/CVE-2022-29145), OpenSSL vulnerability [USN-5402-1](https://ubuntu.com/security/notices/USN-5402-1), curl vulnerability [USN-5412-1](https://ubuntu.com/security/notices/USN-5412-1), and OpenLDAP Vulnerability [USN-5424-1](https://ubuntu.com/security/notices/USN-5424-1)
```

Comment:
Metric Collector is its own module with an independent versioning. We do it's release independently. Please remove it from here.

Reply:
We do have S360 alerts for metrics collector base images as well.

I don't mind removing it and doing the release separately -- but we would need to do a metrics collector release soon enough then.

Do you still want me to remove it?

Reply:
Yes. Make it a separate PR.

Reply:
@yophilav I don't mean to throw a wrench in things...but I don't see why we can't update the release notes in this commit even though we release metrics collector separately? Both releases will point to the same commit, I don't see any problem with that.

Reply:
Sure. I don't see harm in doing that. Personally, I'd prefer to have them as 2 separate PRs to keep my thinking straight what is being updated.